### PR TITLE
fix: Links on avatar for org events

### LINKED ISSUE #12890: [CAL-2835] Avatars in an Org event points to the wrong organization.
[CleanShot 2023-12-20 at 08.45.07.mp4](https://uploads.linear.app/e86bf957-d82f-465e-b205-135559f4b623/4e35a185-b775-4f42-aaee-830493d4ba54/839cd2d2-a638-41f9-9741-c45a324cee52)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2835](https://linear.app/calcom/issue/CAL-2835/avatars-in-an-org-event-points-to-the-wrong-organization)</sub>

### PR DESCRIPTION
**Pull request by hariombalhara**
## What does this PR do?

Fixes #12890

Regression from https://github.com/calcom/cal.com/pull/12158/files#diff-b386e7d0341aaeb0cd268b7608c7ea2b5718349f844f3203fa4c422a4f5ab29dR14

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by hariombalhara** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #12892** <a href="https://app.graphite.dev/github/pr/calcom/cal.com/12892?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/calcom/cal.com/12892?utm_source=stack-comment).

**Comment by hariombalhara** [comment]
Would it make sense to write component test(UserAvatarGroupWithOrg.test.tsx) for this bug?

File: packages/lib/domainManager/deploymentServices/vercel.ts [hunk 1]
```diff
@@ -93,7 +93,7 @@ function handleDomainCreationError(error: { code?: string | null; domain?: strin
     return true;
   }
 
-  const errorMessage = `Failed to take action for domain: ${error.domain}`;
+  const errorMessage = `Failed to create domain on Vercel: ${error.domain}`;
```
Comment (hariombalhara) [inline packages/lib/domainManager/deploymentServices/vercel.ts:96]: Improved the error message in the PR as I saw the error while testing and it wasn't clear enough

File: apps/web/components/ui/avatar/UserAvatarGroupWithOrg.tsx [hunk 2]
```diff
@@ -1,35 +1,32 @@
...
 
 type UserAvatarProps = Omit<React.ComponentProps<typeof AvatarGroup>, "items"> & {
-  users: Pick<User, "organizationId" | "name" | "username">[];
+  users: (Pick<User, "organizationId" | "name" | "username"> & { bookerUrl: string })[];
   organization: Pick<Team, "slug" | "name">;
 };
 
 export function UserAvatarGroupWithOrg(props: UserAvatarProps) {
   const { users, organization, ...rest } = props;
-  const orgBranding = useOrgBranding();
```
Comment (hariombalhara) [inline apps/web/components/ui/avatar/UserAvatarGroupWithOrg.tsx:14]: `useOrgBranding` gives the org details for the logged in user and not the organizer being booked.

**Review by emrysal — APPROVED** [review]
I don't like the extra overhead, but currently working on a more permanent fix for that; will approve.

**Pull request merged**